### DISCUSSION TITLE
Version mismatch: NetBox GUI displays v3.5.1 while NetBox Docker release is v3.5.2

### DISCUSSION THREAD

**Discussion by digital-fortress** [netbox-community/netbox#12683] [Help Wanted!]
2023-05-23 06:30 UTC 6ee33e8d7cf3 (v3.5.1)

/projects/netbox-docker$ git checkout release &&
> git pull -p origin release
Already on 'release'
Your branch is up to date with 'origin/release'.
From https://github.com/netbox-community/netbox-docker
 * branch            release    -> FETCH_HEAD
Already up to date.

**Comment by kkthxbye-code** [comment]
The new image was pushed 19 minutes ago (releases happens once per day for netbox-docker). Are you sure you didn't just try to pull the image before it was released?

Also, this discussion should be in the netbox-docker repo. This repo is for netbox itself.

**Comment by digital-fortress** [comment]
I am still having the same problem. My GUI displays 3.5.2 is available while docker says its uptodate.
